modules/api/src/main/java/org/eclipse/osc/modules/api/OrchestratorApi.java

```diff
@@ -31,21 +36,16 @@ public Response fetch(@HeaderParam("ocl") String oclLocation) throws Exception {
     @Path("/health")
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public String health() throws Exception {
-        if (getOrchestrator() == null) {
-            throw new IllegalStateException("Orchestrator service is not ready");
-        }
+    public String health() {
```

Comment:
These changes are not related to logging right ?

Reply:
removed